corehq/apps/es/users.py

```diff
@@ -195,22 +194,10 @@ def user_ids(user_ids):
     return filters.term("_id", list(user_ids))
 
 
-def primary_location(location_id):
-    # by primary location
-    return filters.OR(
-        filters.AND(mobile_users(), filters.term('location_id', location_id)),
-        filters.AND(
-            web_users(),
-            filters.term('domain_memberships.location_id', location_id)
-        ),
-    )
-
-
 def location(location_id):
     # by any assigned-location primary or not
     return filters.OR(
         filters.AND(mobile_users(), filters.term('assigned_location_ids', location_id)),
-        # todo; this actually doesn't get applied since the below field is not indexed
```

Comment:
This is not true anymore, the filter does return webusers as well. Added a test for that https://github.com/dimagi/commcare-hq/pull/33694/files#diff-e9b28d016f1b2ed92a19743a7fd359714d363a993c485dd3fb34ddfac1e17430R125